.git-blame-ignore-revs

```diff
@@ -0,0 +1,3 @@
+# run this command: git config --global blame.ignoreRevsFile .git-blame-ignore-revs
```

Comment:
what is this file for?

Reply:
Removed.

It ignores the commit where I ran the black commit to preserve the git blame. 
It doesn't do anything unless you run the command `git config --global blame.ignoreRevsFile .git-blame-ignore-revs`.